google/cloud/firestore_v1/aggregation.py

```diff
@@ -78,6 +80,10 @@ def get(
                 (Optional[:class:`~google.cloud.firestore_v1.query_profile.ExplainOptions`]):
                 Options to enable query profiling for this query. When set,
                 explain_metrics will be available on the returned generator.
+            read_time (Optional[datetime]): If set, reads documents as they were at the given
+                time. This must be a microsecond precision timestamp within the past one hour,
+                or if Point-in-Time Recovery is enabled, can additionally be a whole minute
+                timestamp within the past 7 days.
```

Comment:
A couple questions:
- how are timezones handled? Do you expect ["aware" or "naive" datetimes?](https://docs.python.org/3/library/datetime.html#aware-and-naive-objects)
- I just did a quick search, but it seems like datetime.datetime also has microsecond precision. Do we need to mention the precision here?